### DISCUSSION TITLE
Split Features Release Request

### DISCUSSION THREAD

**Discussion by stimpy81** [SimonMacer/AnWave#35] [Open Discussion]
Hello, thank you for this wonderful tool!  Defender remover has become an essential item for me, as it's about the only tool that actually does what it says it does!

Would it be possible for an updated Split Features release, containing all your latest fixes and changes from the full version please?

I would also like to request the removal for typing a code just to remove Defender.  I find a warning more than enough and being forced to type a complex code is just unnecessary and over the top.

Maybe not everyone agrees, but I just wanted to throw it out there for discussion.

Many thanks for your fantastic tools!

**Comment by SimonMacer** [comment]
> Hello, thank you for this wonderful tool! Defender remover has become an essential item for me, as it's about the only tool that actually does what it says it does!
> 
> Would it be possible for an updated Split Features release, containing all your latest fixes and changes from the full version please?
> 
> I would also like to request the removal for typing a code just to remove Defender. I find a warning more than enough and being forced to type a complex code is just unnecessary and over the top.
> 
> Maybe not everyone agrees, but I just wanted to throw it out there for discussion.
> 
> Many thanks for your fantastic tools!

The Microsoft Defender Uninstaller standalone updated to 1.3.25.342 today.
https://github.com/SimonMacer/AnWave/releases/tag/AnWave-Split

**Reply by stimpy81** [reply]
Thank you so much Simon!
